builder/vmware/common/step_register.go

```diff
@@ -62,14 +63,18 @@ func (s *StepRegister) Cleanup(state multistep.StateBag) {
 		if s.SkipExport && !cancelled && !halted {
 			ui.Say("Removing virtual machine from inventory...")
 			if err := remoteDriver.Unregister(s.registeredPath); err != nil {
-				ui.Errorf("error removing virtual machine: %s", err)
+				err = fmt.Errorf("error removing virtual machine from inventory: %v", err)
+				state.Put("error", err)
+				ui.Errorf(err.Error())
```

Comment:
We should probably return after this? Especially considering we've put some `error` in the state, so later on if some other operation fails, it'll get overwritten

Reply:
Although this is cleanup, is the `state` still useful at this point? I would think printing out the error and telling the user that they'll probably need to perform some manual operations to clean this up is enough

Reply:
Refactored in cc75026.